deploy/kubernetes/v0.1.0/controller.yaml

```diff
@@ -0,0 +1,70 @@
+kind: StatefulSet
```

Comment:
I think keeping deployment specs per version is going to get difficult to maintain.  The approach I take is to tag the repo with the version, and if you want an older version, you need to get the spec from that tag.